flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormatTest.java

```diff
@@ -162,11 +162,11 @@ public void testJDBCInputFormatWithParallelismAndNumericColumnSplitting() throws
 				if (next == null) {
 					break;
 				}
-				if(next.getField(0)!=null) { Assert.assertEquals("Field 0 should be int", Integer.class, next.getField(0).getClass());}
-				if(next.getField(1)!=null) { Assert.assertEquals("Field 1 should be String", String.class, next.getField(1).getClass());}
-				if(next.getField(2)!=null) { Assert.assertEquals("Field 2 should be String", String.class, next.getField(2).getClass());}
-				if(next.getField(3)!=null) { Assert.assertEquals("Field 3 should be float", Double.class, next.getField(3).getClass());}
-				if(next.getField(4)!=null) { Assert.assertEquals("Field 4 should be int", Integer.class, next.getField(4).getClass());}
+				if (next.getField(0) != null) { Assert.assertEquals("Field 0 should be int", Integer.class, next.getField(0).getClass());}
```

Comment:
Sorry, I don't know anything about eclipse. The easiest way to check the checkstyle is to run `mvn verify -DskipTests` for the module that you're changing.